src/cmd/src/flownode.rs

```diff
@@ -355,7 +355,8 @@ impl StartCommand {
         .await
         .context(StartFlownodeSnafu)?;
         flownode
-            .flow_worker_manager()
+            .flow_engine()
+            .streaming_engine()
             .set_frontend_invoker(invoker)
```

Comment:
Are we creating a circular reference?

Reply:
> Are we creating a circular reference?

Yes that's a known one, will refactor later with `Weak`